validation/src/test/scala/hmda/validation/rules/lar/validity/V335Spec.scala

```diff
@@ -0,0 +1,36 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Loan, LoanApplicationRegister }
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V335Spec extends LarEditCheckSpec {
+  property("Succeeds when property type is not 3") {
+    forAll(larGen) { lar =>
+      whenever(lar.loan.propertyType != 3) {
```

Comment:
Resolved, but adding a comment here for future readers.

The LARs generated in `LarGenerators.scala` are not supposed to pass all the validations, only the formatting checks.  So, a generated LAR might have the property type 3 and a numeric income (which is checked in a test later in this file).  In this test, we want to make sure the conditional statement works at filtering out all cases where `property type != 3`